flink-core/src/test/java/org/apache/flink/api/java/typeutils/runtime/PojoSubclassSerializerTest.java

```diff
@@ -30,7 +30,7 @@
 import java.util.Random;
 
 /** A test for the {@link PojoSerializer}. */
-public class PojoSubclassSerializerTest
+class PojoSubclassSerializerTest
```

Comment:
Probably it's better also to make use of `import org.junit.jupiter.api.Test` since there are some methods annotated with `@Test`